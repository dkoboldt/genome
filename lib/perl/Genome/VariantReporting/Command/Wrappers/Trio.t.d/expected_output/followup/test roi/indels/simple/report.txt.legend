Headers
chromosome_name	Chromosome: an identifier from the reference genome or an angle-bracketed ID String ("<ID>") pointing to a contig in the assembly file
start	The start position of the variant. One-based
stop	The end position of the variant. One-based, inclusive
reference	The reference allele at this position
variant	The variant called at this position
variant_type	The type of variant: snp, ins (insertion), del (deletion), other
transcript_name	Ensembl stable ID of feature
trv_type	Consequence type of this variation
amino_acid_change	The HGVS protein sequence name
default_gene_name	The gene symbol
ensembl_gene_id	Ensembl stable ID of affected gene
Filters
homopolymer	HOMOPOLYMERS	Filter out indels hitting homopolymers
ft-keep	FTPASS	FT value for sample TEST-patient1-somval_tumor2 is one of [PASS,.]
contains-tag	CONTAINS_TAG_ON_TARGET	INFO field for entry contains tag ON_TARGET
