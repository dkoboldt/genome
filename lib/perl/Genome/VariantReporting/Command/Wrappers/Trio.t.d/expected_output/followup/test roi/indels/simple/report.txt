chromosome_name	start	stop	reference	variant	variant_type	transcript_name	trv_type	amino_acid_change	default_gene_name	ensembl_gene_id
